build/org.eclipse.pde.build/src/org/eclipse/pde/internal/build/site/PDEState.java

```diff
@@ -496,7 +496,9 @@ public static String getSystemPackages(IExecutionEnvironment environment, Proper
 			systemPackages = profileProperties.getProperty(Constants.FRAMEWORK_SYSTEMPACKAGES);
```

Comment:
"use the those its profile" is not understandable

Reply:
Indeed. Should be fixed now.